pages/search.js

```diff
@@ -44,9 +45,14 @@ export default function Search({ users }) {
 
     if (value.length >= 3) {
       setThreeOrMore(true);
-      results = users.filter((user) =>
-        user.name.toLowerCase().includes(value.toLowerCase())
-      );
+      results = users.filter((user) => {
+        if (user.name.toLowerCase().includes(value.toLowerCase())) return true;
+
+        let tag = user.tags?.find((tag) =>
+          tag.toLowerCase().includes(value.toLowerCase())
+        );
+        if (tag) return true;
```

Comment:
Please use the same coding style as the rest of the project 
```suggestion
        if (tag) {
           return true;
        }
```
```